tests/test_generate_report_excels.py

```
@@ -0,0 +1,346 @@
+import unittest
+from pytest import raises, mark, fixture
+import re
+import argparse
+
+import pytest
+
+sys.path.insert(0, os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '..')))
+
+from generate_report_excels import parse_args, \
+    log_start_time, setup_api, get_audit_logs, \
```

Ruff (0.8.2)

`generate_report_excels.setup_api` imported but unused

Remove unused import

(F401)